app/javascript/flavours/glitch/locales/ru.json

```diff
@@ -1,4 +1,6 @@
 {
+  "about.fork_disclaimer": "Glitch-soc - это бесплатное программное обеспечение с открытым исходным кодом, обращенное от Mastodon.",
+  "account.add_account_note": "Добавить заметку для @{name}<a rel=\"me\" href=\"https://infosec.exchange/@Marusy431\">Mastodon</a>",
```

Comment:
?

Reply:
Well-spotted. I don't know if it was a manipulation error on the translator's end or if it was malicious, but I dropped the extra link in Crowdin.